test/inductor/test_flex_attention.py

```diff
@@ -382,7 +382,7 @@ def batch_reserve(paged_attention: PagedAttention, target_seq_len: Tensor):
         )
 
 
-@large_tensor_test_class("2GB", device="cuda")
+@large_tensor_test_class("2GB", device=test_device[0])
```

Comment:
Can you actually remove these, I changed default test sizes so that we use less memory

Reply:
Hi @drisspg! Thanks for the review.

 Do you mean remove all use of `large_tensor_test_class` func in this file?

Reply:
Yeah you can remove all of these

Reply:
done